This is the updated version after kunal's version. With MD5 hashing

kmskrishna opened the pull request.

kunaaljain committed:
Unalign letters, minor tweaks, added updated image

kmskrishna committed:
Added MD5 HASHING and it also includes kunals changes also

kunaaljain commented:
Can't merge it. It has conflicts as the changes i have made are already committed . Remove my commit from pull request and create new one.

kmskrishna commented:
Leave it then. The files in my github page are updated i removed all the conflicts. I took your changed code and updated the md5 hash functions which i wrote.

It is showing conflicts resolve it if they are minor changes. or else leave it.

kunaaljain commented:
never mind. I fixed the conflicts :)
